app/controllers/bibliographic_controller.rb

```diff
@@ -295,17 +295,37 @@ def bib_id_param
       sanitize(params[:bib_id])
     end
 
+    def needs_locator_call_no(location)
+      # For now only Firestore items need this.
+      return true if (location || "").start_with?("firestone$")
```

Comment:
From looking at the ticket it seems like we can `return true if location == "firestone$stacks`